src/app-simplefin/app-simplefin.js

```diff
@@ -50,14 +50,19 @@ app.post(
     const startDate = new Date(now.getFullYear(), now.getMonth(), 1);
     const endDate = new Date(now.getFullYear(), now.getMonth() + 1, 1);
 
-    const accounts = await getAccounts(accessKey, startDate, endDate);
+    try {
+      const accounts = await getAccounts(accessKey, startDate, endDate);
 
-    res.send({
-      status: 'ok',
-      data: {
-        accounts: accounts.accounts,
-      },
-    });
+      res.send({
+        status: 'ok',
+        data: {
+          accounts: accounts.accounts,
+        },
+      });
+    } catch (e) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you also add a `console.log` in these catch blocks? So we have a bit more debug information if-need-be.

Reply:
Done